platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/Template.java

```diff
@@ -40,6 +40,6 @@ public void setFileName(String fileName) {
 
     @Override
     public String filePath() {
-        return String.format("/%s/%s", language, fileName);
+        return String.format("%s", fileName);
```

Comment:
Also, `filePath` is not used in any of the other changes. Is this a related change?